### PR TITLE
Adding an invitation to the discord server

### PR DESCRIPTION
**Pull request by Zaptyp**
Adding an invitation to the discord server into README.md!!!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,11 @@
 🌋 Unofficial browser VULCAN UONET+ client for both students and their parents
 
 ![GitHub Workflow Status](https://github.com/wulkanowy/wulkanowy-web/workflows/Python%20application/badge.svg)
-[![Discord](https://img.shields.io/discord/390889354199040011.svg?color=#33CD56)](https://discord.gg/vccAQBr)
+
+## Come to our discord server!!!
+[![Discord1](https://i.imgur.com/3ljTQWy.png)](https://discord.gg/5qsEujZMdp)
```
Comment (mklkj) [inline README.md:7]: Please, use the official invitation link from our home page

**Review by mklkj — DISMISSED** [review]

**Review by Pengwius — COMMENTED** [review]
Hey, discord invation link already exists in README.

File: README.md [hunk 2]
```diff
@@ -2,7 +2,9 @@
 🌋 Unofficial browser VULCAN UONET+ client for both students and their parents
 
 ![GitHub Workflow Status](https://github.com/wulkanowy/wulkanowy-web/workflows/Python%20application/badge.svg)
-[![Discord](https://img.shields.io/discord/390889354199040011.svg?color=#33CD56)](https://discord.gg/vccAQBr)
+
+## Come to our discord server!!!
```
Comment (MRmlik12) [inline README.md]: Please, remove 3 exclamation marks

File: README.md (hunk 1 above)
Comment (Zaptyp) [reply README.md:7]: Ok

**Review by MRmlik12 — APPROVED** [review]
Approved!

File: README.md (hunk 2 above)
Comment (Zaptyp) [reply README.md]: Ok

**Pull request merged**